piksi_tools/console/console.py

```diff
@@ -159,6 +168,7 @@ class SwiftConsole(HasTraits):
   system_monitor_view = Instance(SystemMonitorView)
   settings_view = Instance(SettingsView)
   update_view = Instance(UpdateView)
+  log_level_filter = Enum(list(SYSLOG_LEVELS.itervalues()))
```

Comment:
Here I get the possible log levels from a dictionary in the output_list class and use Traits ENUM to handle the gui and selection.